src/Core/src/WindowExtensions.cs

```diff
@@ -30,10 +33,41 @@ public static partial class WindowExtensions
 #endif
 		}
 
+		public static float GetDisplayDensity(this IWindow window)
```

Comment:
Remove this extention Method

use the "GetWindow" extention method against the IGridLayout that's inside GridLayoutManager

That will give you an IWindow

You can retrieve the RequestDisplayDensity from that

If the window is null just use 1 for the DisplayDensity